### PR TITLE
feat: add todomvc example for use in updated scaffolding

### PR DESCRIPTION
**Pull request by panzarino**
Adds a modified version of todomvc for use in updated scaffolding

Depends on changes in https://github.com/cypress-io/cypress/pull/15826

### CODE REVIEW

File: cypress/integration/1-getting-started/todo.spec.js [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+    // we can chain both of these assertions together into a single statement.
+    cy.get('.todo-list li')
+      .should('have.length', 3)
+      .last()
+      .should('have.text', newItem)
+  })
+
+  it('can check off an item as completed', () => {
+    // In addition to using the `get` command to get an element by selector,
+    // we can also use the `contains` command to get an element by it's contents.
```
Comment (jennifer-shehane) [inline cypress/integration/1-getting-started/todo.spec.js:62]: 😄 I do this about 50% of the time I write its

```suggestion
    // we can also use the `contains` command to get an element by its contents.
```

File: cypress/integration/1-getting-started/todo.spec.js [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+    // However, this will yield the lowest-level element that contains the text
+    // which is a <label> in this case.
+    // We want to check off the item, so we actually need to select the <input> element
+    // that is linked to that <label>. In order to do this,
+    // we can traverse a level up the dom to the parent element. From there, we can `find`
+    // a child checkbox <input> element and use the `check` command to check it.
```
Comment (jennifer-shehane) [inline cypress/integration/1-getting-started/todo.spec.js:68]: This is a bit wordy. 
```suggestion
    // However, this will yield the <label>, which is lowest-level element that contains the text.
    // In order to check the item, we'll find the <input> element for this <label>
    // by traversing up the dom to the parent element. From there, we can `find`
    // the child checkbox <input> element and use the `check` command to check it.
```

File: cypress/integration/1-getting-started/todo.spec.js [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+    cy.contains('Pay electric bill')
+      .parent()
+      .find('input[type=checkbox]')
+      .check()
+
+    // Now that we've checked the button, we can go ahead and make sure
+    // that the list element is now marked as completed.
+    // Again we'll use contains to find the <label> element and then use the `parents` command
```
Comment (jennifer-shehane) [inline cypress/integration/1-getting-started/todo.spec.js:76]:
```suggestion
    // Again we'll use 'contains' to find the <label> element and then use the `parents` command
```

**Review by jennifer-shehane — CHANGES_REQUESTED** [review]
Just a few little things. Otherwise it looks good.

Also, this probably shouldn't be on you, but this repo should be running against Cypress 7.3.0, since we want to merge it up to `cypress` develop. This PR looks like a mess https://github.com/cypress-io/cypress-example-kitchensink/pull/480

**Comment by panzarino** [comment]
@jennifer-shehane Added suggestions - PR for `on` link is here https://github.com/cypress-io/cypress-services/pull/3839

**Review by jennifer-shehane — APPROVED** [review]
